Dockerfile

```diff
@@ -1,29 +1,39 @@
-FROM ubuntu:bionic
+FROM php:7.3-apache
 
-# TODO: think through optimizations here once this is working.
-RUN    apt-get update \
-    && apt --yes install software-properties-common \
-    && apt-add-repository --yes --update ppa:ansible/ansible \
-    && apt --yes install git ansible
+# Extensions we know LF requires/desires:
+# curl - already provided in image
+# mbstring - already provided in image
+# xdebug
+# intl
+# gd
+# mongodb
+#
+# see https://github.com/mlocati/docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions gd xdebug mongodb curl intl
+RUN install-php-extensions @composer
 
-# make a working area, an "app" area
+# install Language Forge APT dependencies
+RUN apt-get update && apt-get -y install p7zip-full unzip
+
+# Pull in Language Forge git repo
 RUN mkdir -p /data
 WORKDIR /data
-
-# go ahead and pull local filesystem in so we can take care of more manual configs
-# TODO: need to optimize this by only copying things that are needed for the set up
 COPY . /data
 
-# not sure whether to handle this outside of this image, i.e., tell user to git clone ... --recurse-submodules, or just do it here to make sure the submodules are guaranteed to be here for ansible scripts...
-RUN git submodule update --init --recursive
 
-# time to run the ansible scripts
-WORKDIR /data/deploy
 
-# TODO: need to make a distinction between environment setup commands and those commands meant for running the services.
-# TODO: left off the -K otherwise this command becomes interactive (password requested)
-# RUN ansible-playbook playbook_bionic.yml --limit localhost
+# apt packages to investigate further
+#   - default-jre-headless
+#   - lfmerge
+#   - nodejs
+#   - postfix
+#   - git
+#
+#
+#
 
-# troubleshooting latest failure...
-# `docker-compose run --rm lf bash` will get you into container
-# `ansible-playbook playbook_bionic.yml --limit localhost` (with or without -K not relevant afaik at this point but bash-ing into the container will provide the opportunity to run with the -K (just enter at prompt))
+# it feels like these should run in separate containers:
+# nodejs
+# postfix (for sending email)
```

Comment:
absolutely...we'd want to investigate the existing postfix images out there but a quick search and https://hub.docker.com/r/juanluisbaptiste/postfix already looks promising.